composer.json

```diff
@@ -18,10 +18,17 @@
         "ext-gd": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0",
+        "symfony/phpunit-bridge": "^5.2",
         "friendsofphp/php-cs-fixer": "~2.17"
     },
-    "config": {
-        "bin-dir": "bin"
+    "scripts": {
+        "cs": "php-cs-fixer fix --verbose --dry-run --diff",
+        "fix": "php-cs-fixer fix --verbose --diff",
```

Comment:
Could these two be renamed to be more clear?

Like `cs-check` and `cs-fix` perhaps?